api/client/info.go

```diff
@@ -53,6 +53,14 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 	}
 	fmt.Fprintf(cli.out, "\n")
 
+	if len(info.Plugins.Authorization) != 0 {
+		fmt.Fprintf(cli.out, " Authorization:")
+		for _, driver := range info.Plugins.Authorization {
```

Comment:
[`strings.Join`](https://golang.org/pkg/strings/#Join) and a single call to `fmt.Fprintf`?